mlflow/langchain/__init__.py

```diff
@@ -359,7 +366,7 @@ def load_retriever(persist_directory):
         **model_data_kwargs,
     )
 
-    if Version(langchain.__version__) >= Version("0.0.311"):
+    if Version(langchain.__version__) >= Version("0.0.311") and not resources:
         if databricks_resources := _detect_databricks_dependencies(lc_model):
             serialized_databricks_resources = _ResourceBuilder.from_resources(databricks_resources)
             mlflow_model.resources = serialized_databricks_resources
```

Comment:
I might be missing something here, but dont we need an else statement to set mlflow_model.resources to the resources passed in from the method

Reply:
Model.log already has the logic to do that :)